packages/loot-core/src/server/accounts/rules.ts

```diff
@@ -726,10 +726,10 @@ export function execActions(actions: Action[], transaction) {
 }
 
 export class Rule {
-  actions;
-  conditions;
+  actions: Action[];
+  conditions: Condition[];
   conditionsOp;
-  id;
+  id?: string;
   stage;
```

Comment:
From looking through the code-base, I _think_ this would be the type?
```suggestion
  stage: 'pre' | 'default' | 'post';
```

Reply:
I though this was `'pre' | 'post' | null`

Reply:
Oh you're right based on my reading of the logic. I think the place I looked is actually a bug. Mind updating the logic [here](https://github.com/actualbudget/actual/blob/master/packages/desktop-client/src/components/accounts/Account.tsx#L1193) to reflect? And maybe good to fix the type [here](https://github.com/actualbudget/actual/blob/master/packages/loot-core/src/types/models/rule.d.ts#L4)